src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+form {
+  position: relative;
+  margin: 20px 7px;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  outline: none;
+}
+
+.input--big {
+  height: 70px;
+  padding: 25px 0 23px 62px;
```

```suggestion
  padding-left: 62px;
```
You need here only left padding